gordo_components/workflow/controller_to_sql/controller_to_sql.py

```diff
@@ -62,21 +60,21 @@ def endpoint_to_machine_data(ep):
     )
 
 
-def watchman_to_sql(
-    watchman_address: str,
+def controller_to_sql(
+    controller_address: str,
     sql_host: str,
     sql_port: int = 5432,
     sql_database: str = "postgres",
     sql_username: str = "postgres",
     sql_password: str = None,
 ):
     """
-    Connects to watchman, downloads all metadata, and proceeds to push them to postgres.
+    Connects to controller, downloads all metadata, and proceeds to push them to postgres.
```

Comment:
This kind of makes it sound like 'all metadata' is on the controller.

Reply:
Resolving this, as it's not irrelevant, just that it is given the new direction this PR will go (ie. no longer querying controller)